### ISSUE TITLE
It messes up http://necolas.github.com/normalize.css/

### ISSUE THREAD

**Issue by nhoizey** [nicooprat/Css2Less#4]

**Comment by nicooprat** [comment]
Selectors reordering is a known issue.

But actually, for this kind of files, I would recommend to you to either not parse it to LESS (you won't edit it anyway, right ?), or @import the "normalize.css" file into your .less file ; which is probably better than just pasting it at the beginning of your styles.

So it's maybe not a "real bug".

**Comment by nhoizey** [comment]
I agree this specific CSS can be used without LESS conversion, it was just an example of a bug…

**Comment by nicooprat** [comment]
You're right, thanks for pointing it out.
